config.js

```diff
@@ -29,7 +29,7 @@ var config = {};
  * Default log level. Can be one of: 'DEBUG', 'INFO', 'WARN', 'ERROR', 'FATAL'
  * @type {string}
  */
-config.logLevel = 'INFO';
+config.logLevel = 'DEBUG';
```

Comment:
Please rollback changes in this file. Except in the case of very good reasons (is there any?) reference configuration here shouldn't be changed.

Reply:
It's a mistake, sorry.

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad

Reply:
Are you sure? Looking to the diff tab I'm still seeing changes in config.js file ...

Reply:
I've reviewed the commit and it's corrected.
https://github.com/fiqare-emergya-dev/cdti-fiqare-perseo-fe-src/blob/fiqare-improvements-1/config.js

Reply:
With regards to this particular change, that's right, it has been rolled-back.

However, the PR stills containing changes in config.js. I see the following in the "Diff" tab:

Reply:
Fixed in ff07d1070d5096cdbc0e359130c6a04514f87ea7